lambdas/checkpassport/src/main/java/uk/gov/di/ipv/cri/passport/checkpassport/services/dvad/DvadThirdPartyAPIService.java

```diff
@@ -192,6 +180,42 @@ private void assertNoErrorsSetInGraphQLResponse(GraphQLAPIResponse graphQLAPIRes
         }
     }
 
+    private String getErrorLine(Errors error) {
+
+        // There are multiple Error response formats with optional fields and overloaded types
+
+        String messageSegment = String.format("message %s, ", error.getMessage());
+
+        List<String> path = error.getPath();
+        String pathSegment = path == null || path.isEmpty() ? "" : String.format("path %s,", path);
+
+        List<Locations> locations = error.getLocations();
+        String locationsSegment =
+                locations == null || locations.isEmpty()
+                        ? ""
+                        : String.format("locations %s, ", locations);
+
+        Extensions extensions = error.getExtensions();
+
+        String errorCode = extensions.getErrorCode();
+        String errorCodeSegment =
+                errorCode == null ? "" : String.format("errorCode %s, ", errorCode);
+
+        // Classification displayed last as its value may single string or a complex object (as a
+        // string)
+        String classification = extensions.getClassification().toString();
```

Comment:
Are we sure classification will always be present we null check the rest but not this